_posts/2019/2019-03-13-my-switch-from-js-to-ruby.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: My switch from JavaScript to Ruby
+date: 2019-03-13
+header_image: public/my-switch-from-js-to-ruby.jpg
+header_position: center
+category: tech-stories
+tags: ["javascript", "ruby", "development"]
+authors: ["German"]
+about_authors: ["gsanemeterio"]
+---
+
+Hello, my name is German, I am currently part of ePages as software engineer with ruby as language, although before I worked with Angular.
```

Comment:
As people read your name in the front matter already, I suggest to just state that you've started as a software developer in January.